### PR TITLE
Add updated docs and content for JourneyMap 5.8.5

### PR DESCRIPTION
**Pull request by stantios**
Description: This PR includes all the options and changes that happened in 5.8.5, this also includes a few tweaks that I added myself which are mostly QoL.

This also closes https://github.com/TeamJM/journeymap-docs/issues/6

Note: This PR should not be merged until the deploy.yml is changed to deploy a new version called 5.8.5 and set that as the default version.

### CODE REVIEW

File: docs/en/Server Docs/settings/global-properties.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| Animal Radar            | If players can see animals on the map.                                                                                                                     |
+| Monster/Hostile Radar   | If players can see monsters or hostile entities on the map.                                                                                                |
+
+## **Other Settings**
+
+The default option for each setting below is marked with **bold** text.
+
+| Setting                    | Options                                           | Description                                                                                                                                              |
+|----------------------------|---------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Expanded Radar             | <ul><li>**All**</li><li>Op</li><li>None</li></ul> | The format of your coordinates, as shown in an info slot                                                                                                 |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/global-properties.md]: `The format of your coordinates, as shown in an info slot`
is wrong. 
Expanded radar is "Let's the players see other players on the map outside of render range, anywhere in their current dimension."

File: docs/en/Server Docs/settings/global-properties.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+The Global Properties category contains settings that affect the server side behaviour of the mod. These are the default properties for the server.
+
+![Global-Properties](../../img/settings/server/global-properties.png){: .center}
+
+## **Toggles**
+
+| Toggle                  | Description                                                                                                                                                |
+|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Allow JourneyMap        | Whether to allow journeymap to function for non-ops.                                                                                                       |
+| World ID                | Enabling will change the save directory for this servers mapping data. If disabled and then enabling on an active server will reset all user mapping data. |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/global-properties.md]: You missed some of the ingame tooltip text for World ID. You need need the text that describes what it is.

File: docs/en/Server Docs/settings/dimension-overworld.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+The Dimension: minecraft:overworld category contains settings that will be the default settings for the overworld dimension.
+
+![Dimension-Minecraft-Overworld](../../img/settings/server/dimension-minecraft-overworld.png){: .center}
+
+## **Toggles**
+
+| Toggle                | Description                                                                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enable                | Enabling this dimension will override the global properties for this dimension.                                                                            |
+| Waypoint Teleporting  | Enabling will change the save directory for this servers mapping data. If disabled and then enabling on an active server will reset all user mapping data. |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/dimension-overworld.md]: THis text is wrong also

File: docs/en/Server Docs/settings/dimension-nether.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+The Dimension: minecraft:the_nether category contains settings that will be the default settings for the nether dimension.
+
+![Dimension-Minecraft-Nether](../../img/settings/server/dimension-minecraft-nether.png){: .center}
+
+## **Toggles**
+
+| Toggle                | Description                                                                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enable                | Enabling this dimension will override the global properties for this dimension.                                                                            |
+| Waypoint Teleporting  | Enabling will change the save directory for this servers mapping data. If disabled and then enabling on an active server will reset all user mapping data. |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/dimension-nether.md]: wrong text here

File: docs/en/Server Docs/settings/dimension-end.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+The Dimension: minecraft:the_end category contains settings that will be the default settings for the end dimension.
+
+![Dimension-Minecraft-End](../../img/settings/server/dimension-minecraft-end.png){: .center}
+
+## **Toggles**
+
+| Toggle                | Description                                                                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enable                | Enabling this dimension will override the global properties for this dimension.                                                                            |
+| Waypoint Teleporting  | Enabling will change the save directory for this servers mapping data. If disabled and then enabling on an active server will reset all user mapping data. |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/dimension-end.md]: wrong text here

File: docs/en/Server Docs/settings/default-dimension-properties.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+The Default Dimension Properties category contains settings that will be the default settings for all new dimensions created. These settings can be overridden on a per-dimension basis.
+
+![Default-Dimension-Properties](../../img/settings/server/default-dimension-properties.png){: .center}
+
+## **Toggles**
+
+| Toggle                | Description                                                                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enable                | Enabling this dimension will override the global properties for this dimension.                                                                            |
+| Waypoint Teleporting  | Enabling will change the save directory for this servers mapping data. If disabled and then enabling on an active server will reset all user mapping data. |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/default-dimension-properties.md]: wrong text here

File: docs/en/Server Docs/multiplayer.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+## **Settings**
+
+The Multiplayer section is a section designed to give users control on how they are viewed on other clients using the JourneyMap mod, and how they are displayed on the map. This section is only unlocked when the server is running the JourneyMap mod.
```
Comment (mysticdrew) [inline docs/en/Server Docs/multiplayer.md]: Add here that these settings can also be disabled by the server.

File: docs/en/Server Docs/installing.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+## **Installing**
+
+JourneyMap Server is an optional server-side utility for Forge, Fabric and Bukkit servers. As of JourneyMap 5.1.1, the server-side mod comes in the same jar as the JourneyMap client with Forge and Fabric.
```
Comment (mysticdrew) [inline docs/en/Server Docs/installing.md]: change this to
Installing JourneyMap on the server for Forge or Fabric adds some optional utility for server admins. Note: Forge or Fabric versions can connect to JourneyMap on the server no matter if it is Forge or Fabric.

File: docs/en/Client Docs/waypoints.md [hunk 9]
```diff
@@ -73,6 +73,20 @@ As of JourneyMap 5.2, you can now easily share waypoints and locations with othe
...
+
+**Reload Waypoints**
+
+- Reloads waypoints in the waypoint directory without having to restart.
+- Primarily used when dropping waypoint files into the folder while the game is running.
+
+```
+/jm reload
+```
+The create and delete commands are only available when the server has JourneyMap installed and if the player has op permissions. You can find more information about the add and delete commands [here](https://teamjm.github.io/journeymap-docs/Server%20Docs/waypoints/).
```
Comment (mysticdrew) [inline docs/en/Client Docs/waypoints.md:88]: 👍

File: docs/en/Server Docs/Endpoints/waypoint.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
+## **Waypoints**
+
+The External Waypoint Packet Listener is a feature that allows the server to send waypoints to the client.  This is useful for server admins who want to provide waypoints to players without requiring an addon mod.
+
+The waypoints are stored in the world's waypoint directory which is <code>./minecraft/journeymap/data/mp/{worldname}/waypoints</code>, and are sent to the client whenever the server wants the user to see them. The waypoints are not editable by the player, but they can be deleted if the origin for the waypoint is 'external'. The waypoints are not persistent, so they will not be saved to the client's config file.
```
Comment (mysticdrew) [inline docs/en/Server Docs/Endpoints/waypoint.md]: remove this whole line please. It is not needed, this makes it sound like server admins can put waypoints in that directory and it will automatically send the waypoints. Which is not the case.

File: docs/en/Server Docs/basic-usage.md [hunk 11]
```diff
@@ -13,9 +11,7 @@ If the button does not appear, you may need to log out and back into your server
 
 By default all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file is location: `(server_folder)/configs/journeymap_server.cfg`. 
-
-If you are using Minecraft 1.16+ the config file location: `(server_folder)/world/serverconfig/journeymap_server.cfg`.
+The config file is location: ```(server folder)/journeymap/server/<version>/journeymap.server.minecraft~<world>.config```.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md]: this did not change from 1.16+ revert this change

File: docs/en/Server Docs/basic-usage.md [hunk 12]
```diff
@@ -13,9 +11,7 @@ If the button does not appear, you may need to log out and back into your server
 
 By default all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file is location: `(server_folder)/configs/journeymap_server.cfg`. 
-
-If you are using Minecraft 1.16+ the config file location: `(server_folder)/world/serverconfig/journeymap_server.cfg`.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md:18]: This should be re-added, this is for forge

File: docs/en/Server Docs/basic-usage.md [hunk 13]
```diff
@@ -13,9 +11,7 @@ If the button does not appear, you may need to log out and back into your server
 
 By default all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file is location: `(server_folder)/configs/journeymap_server.cfg`.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md:16]: this is the config location for fabric now.

**Review by mysticdrew — CHANGES_REQUESTED** [review]
Looks like a bunch of copy paste errors and some other things that need to be change.

**Comment by stantios** [comment]
I fixed all the changes you requested and changed the values in deploy.yml to deploy 5.8.5

File: docs/en/Client Docs/settings/advanced.md [hunk 14]
```diff
@@ -18,8 +18,12 @@ The **bold** toggle settings below are enabled by default.
...
+| **Chunk Data Caching**     | Enables or disables chunk caching                                                                                                                       |
+| GL Error Checking          | Enables or disables OpenGL error checking                                                                                                               |
 | **Hide Sneaking Entities** | Whether sneaking/crouching creatures should be hidden                                                                                                   |
+| Hide Spectators            | Whether spectators should be hidden on the radar                                                                                                        |
 | **High Display Quality**   | Uncheck to improve zoom performance and memory usage, but reduce <br>display quality and lower performance of minimap rotation when set to “My Heading” |
+| Multi-World Support        | Enables or disables multi-world support                                                                                                                 |
```
Comment (mysticdrew) [inline docs/en/Client Docs/settings/advanced.md]: Change this one to 
Enables or disables multi-world support - Note: this may not work for all servers.

File: docs/en/Server Docs/basic-usage.md [hunk 15]
```diff
@@ -9,9 +9,11 @@ All Options in the Server Admin Screen have tooltips that explain them in detail
 
 If the button does not appear, you may need to log out and back into your server if you were recently Opped, or the Admin disabled it in the config file. 
 
-By default all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
+By default, all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file is location: ```(server folder)/journeymap/server/<version>/journeymap.server.minecraft~<world>.config```. 
+The config file location for Fabric Servers is: `(server_folder)/configs/journeymap_server.cfg`.
+
+If you are using Minecraft 1.16+ the config file location: `(server_folder)/world/serverconfig/journeymap_server.cfg`.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md]: The config file location for Forge Servers:

File: docs/en/Server Docs/basic-usage.md [hunk 16]
```diff
@@ -9,9 +9,11 @@ All Options in the Server Admin Screen have tooltips that explain them in detail
 
 If the button does not appear, you may need to log out and back into your server if you were recently Opped, or the Admin disabled it in the config file. 
 
-By default all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
+By default, all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file is location: ```(server folder)/journeymap/server/<version>/journeymap.server.minecraft~<world>.config```. 
+The config file location for Fabric Servers is: `(server_folder)/configs/journeymap_server.cfg`.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md]: Remove `is` please

File: docs/en/Server Docs/settings/default-dimension-properties.md [hunk 17]
```diff
@@ -1,16 +1,17 @@
...
-| Player Radar          | If players can see other players on the map.                                                                                                               |
-| Villager Radar        | If players can see villagers on the map.                                                                                                                   |
-| Animal Radar          | If players can see animals on the map.                                                                                                                     |
-| Monster/Hostile Radar | If players can see monsters or hostile entities on the map.                                                                                                |
+| Toggle                | Description                                                                     |
+|-----------------------|---------------------------------------------------------------------------------|
+| Enable                | Enabling this dimension will override the global properties for this dimension. |
+| Waypoint Teleporting  | Allows or prevents teleport teleporting via waypoint manager.                   |
```
Comment (mysticdrew) [inline docs/en/Server Docs/settings/default-dimension-properties.md]: Sorry, I should have mentioned this earlier. 
This setting also effects teleporting from the fullscreen map right click context menus. 
Change for all waypoint teleporting option texts. 
"Allows or prevents teleport teleporting via waypoint manager and Fullscreen context menu."

**Review by mysticdrew — CHANGES_REQUESTED** [review]
A few more changes.

File: docs/en/Server Docs/basic-usage.md [hunk 18]
```diff
@@ -11,9 +11,9 @@ If the button does not appear, you may need to log out and back into your server
 
 By default, all Opped users have access to the Server Admin screen. This can be changed via a config file on the server. 
 
-The config file location for Fabric Servers is: `(server_folder)/configs/journeymap_server.cfg`.
+The config file location for Fabric Servers: `(server_folder)/configs/journeymap_server.cfg`.
 
-If you are using Minecraft 1.16+ the config file location: `(server_folder)/world/serverconfig/journeymap_server.cfg`.
+If you are using Minecraft 1.16+, the config file location for Forge servers: `(server_folder)/world/serverconfig/journeymap_server.cfg`.
```
Comment (mysticdrew) [inline docs/en/Server Docs/basic-usage.md]: "The config file location for Forge Servers:"

the other text is not need here since 5.8.x is only available for 1.16.

**Review by mysticdrew — CHANGES_REQUESTED** [review]
one more change

**Review by mysticdrew — APPROVED** [review]
